lldb/include/lldb/Interpreter/CommandInterpreter.h

```diff
@@ -766,6 +768,12 @@ class CommandInterpreter : public Broadcaster,
   CommandUsageMap m_command_usages;
 
   StreamString m_transcript_stream;
+
+  /// Contains a list of handled commands, output and error. Each element in
+  /// the list is a dictionary with three keys: "command" (string), "output"
+  /// (list of strings) and optionally "error" (list of strings). Each string
+  /// in "output" and "error" is a line (without EOL characters).
+  StructuredData::ArraySP m_transcript_structured;
```

Comment:
nit: No need to put `structured` in the variable name.

Reply:
There is another variable right above this one, called `m_transcript_stream`. So I added the `structured` suffix to emphasize the difference.